Documentation/deploy-documentation.sh

```diff
@@ -1,37 +1,22 @@
 #!/bin/bash
-set -e # Exit with nonzero exit code if anything fails
+jazzy --config .jazzy.json
 
-# Pull requests and commits to other branches shouldn't try to deploy
-if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
-    echo "Skipping documentation deploy."
-    exit 0
-fi
+github_user=$1
+github_password=$2
 
-cd Documentation
-virtualenv ./virtualenv
-source virtualenv/bin/activate
-pip install -r requirements.txt
-
-make clean
-make html
-
-cd build
+cd Documentation/
 rm -rf gh-pages
-git clone -b gh-pages https://${DOC_PUSH_TOKEN}@github.com/gini/gini-vision-lib-ios.git gh-pages
+git clone -b gh-pages https://"$github_user":"$github_password"@github.com/gini/gini-vision-lib-ios.git gh-pages
```

Comment:
Doesn't the Jenkins machine have an SSH key for Github? It might be possible to use SSH here and skip those credentials.

Reply:
I supposed that It doesn't, since it the other repo it was not used. I think it is fine to use both user and password, so we don't have to worry about setting up the ssh key.